composer.json

```diff
@@ -19,7 +19,7 @@
 	"require": {
 		"php": ">=7.1",
 		"kdyby/strict-objects": "^2.0",
-		"latte/latte": "^2.5.2 || ^3.0",
+		"latte/latte": "^2.5.2 <= 2.10.1",
```

Comment:
why?

Reply:
There is a compatibility issue with latte/latte:dev-master. There are BCs that are not compatible with nette/application. Specifically: Nette\Bridges\ApplicationDI\LatteExtension. There is probably a backlog from nette developers. I hope they fix it soon. I'm helpless at the moment.

Reply:
it seems the next latte version will be 3.0 though so this shouldn't be necessary here

Reply:
I forgot about the --prefer-stable local test. Now there is the original dependence.